Mainnet/AddressMapper/AddressMapper/AddressMapper.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using Stratis.SmartContracts;
+
+public class AddressMapper : SmartContract
+{
+    public Address GetSecondaryAddress(Address primary) => PersistentState.GetAddress($"SecondaryAddress:{primary}");
+    private void SetSecondaryAddress(Address primary, Address secondary) => PersistentState.SetAddress($"SecondaryAddress:{primary}", secondary);
+
+    private MappingInfo GetMappingInfo(Address secondary) => PersistentState.GetStruct<MappingInfo>($"MappingInfo:{secondary}");
+    private void SetMappingInfo(Address secondary, MappingInfo value) => PersistentState.SetStruct($"MappingInfo:{secondary}", value);
+    
+    private void ClearMappingInfo(Address secondary) => PersistentState.Clear($"MappingInfo:{secondary}");
+
+    public Address Owner
+    {
+        get => PersistentState.GetAddress(nameof(Owner));
+        private set => PersistentState.SetAddress(nameof(Owner), value);
+    }
+
+    public AddressMapper(ISmartContractState smartContractState, Address Owner) : base(smartContractState)
+    {
+        this.Owner = Owner;
```

Comment:
Just verifying, the intention here is to allow any owner by sending an address on contract creation and not make the owner `Message.Sender` by default?

Reply:
Yes it is intentionally designed this way